apps/livreGUI/Controller.cpp

```diff
@@ -21,238 +21,102 @@
  */
```

Comment:
Fair enough to do through the signal slots, which means every event somehow should be exported as a signal in C++ code when needed. As I have said, if this is the way we want that, I have no objections.